.github/workflows/coverage.yml

```diff
@@ -20,20 +25,48 @@ jobs:
   Coveralls:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      # checkout comparison branch
```

Comment:
oh are you thinking of running them in parallel?

Reply:
We only run coveralls once in the end

Reply:
yea, just thought it may be good to to run them in parallel so both branch need to install, build, and run `yext/slapshot-reusable-workflows/get-coverage-percent`.

Reply:
no get-coverage-percent manually reads the lcov it doesn't send anything to coveralls